lib/hyrax/configuration.rb

```diff
@@ -373,6 +373,11 @@ def subject_prefix
       @subject_prefix ||= "Contact form:"
     end
 
+    attr_writer :extract_full_text
+    def extract_full_text?
+      @extract_full_text ||= false
```

Comment:
For backwards compatibility the default should be true, right?  I don't think you can use the standard memorization idiom with booleans (consider what happens if `@extract_full_text = false`)